submissions/aderonkeOdebunmi/IntroductiontoHTML/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charsset="UTF-8">
+<title>EveryQueen Join/Log In</title>
+</head>
+<body bgcolor="Gold"></body>
```

Comment:
@a-Ronke remove the styling here.
You are suppose to work on a product page not a contact form. Kindly update.